upgrade_versions.py

```diff
@@ -186,13 +129,12 @@ def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
         if log:
             assert user_name is not None
             assert user_email is not None
-            rpm_bump_spec(pkg.specfile, log, user_name, user_email)
+            pkg.spec_file.bump_release(log, user_name, user_email)
 
     def _read_spec(self):
```

Comment:
I don't think this method makes sense anymore. Just make the calls directly to `self.pkg.version`/`release` or move these attributions to the constructor.